src/main/kotlin/no/nav/familie/ef/mottak/task/ArkiverSøknadTask.kt

```diff
@@ -9,6 +9,7 @@ import no.nav.familie.prosessering.domene.Task
 import no.nav.familie.prosessering.domene.TaskRepository
 import org.springframework.data.repository.findByIdOrNull
 import org.springframework.stereotype.Service
+import java.util.*
```

Comment:
Jeg tror det er intellij som ikke alltid likes oss, med att den ikke får med seg familie-settings? :/

Reply:
Ja, ser sånn ut, føler jeg har satt familie-settings på nytt i intellij x-antall ganger nå :/
Rettet det.

Reply:
Jeg hadde problemet hele tiden tidligere, husker ikke helt hva jeg gjorde sen - kjør 2020.03
Tror jeg slettet familie-settings fra import, lukket alle vinduer, en etter en - sånn att man kun har en åpen, lukket den også. Åpnet intellij på nytt, importerte settings og sen åpnet de andre intellij-prosjektene som jeg hadde åpne tidligere :P Tror det hjalp meg
Veldig rart att intellij sliter så mye med det..

Reply:
Okay, takk for tips! Gjør et forsøk det :)